sdk/python/tests/test_ArrayHelpers.py

```diff
@@ -324,4 +391,19 @@ def test_write_variable_size_elements_writes_all_elements_and_aligns(self):
 		# * 113 - aligned up to 106: [113, 0, 0, 0]
 		self.assertEqual(bytes([101, 0, 0, 0, 104, 107, 0, 110, 0, 0, 113, 0, 0, 0]), output)
 
+	def test_write_variable_size_elements_ex_last_writes_all_elements_and_aligns_ex_last(self):
+		# Arrange:
+		context = self.ElementsTestContext()
+
+		# Act:
+		output = ArrayHelpers.write_variable_size_elements(context.elements, 4, exclude_last=True)
+
+		# Assert: notice that alignment is calculated from reported size, not serialized size
+		# * 101 - aligned up to 104: [101, 0, 0, 0]
+		# * 104 - aligned up to 104: [104]
+		# * 107 - aligned up to 108: [107, 0]
+		# * 110 - aligned up to 102: [110, 0, 0]
+		# * 113 - NOT aligned: [113]
```

Comment:
i noticed numbers are wrong, can you update ?

Reply:
I'm not quite sure what the comment was trying to say, so changed it a bit